dataland-frontend/src/components/forms/parts/elements/basic/MultiSelectFormElementBindData.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <MultiSelect
+    v-model="selections"
+    :options="options"
+    :placeholder="placeholder"
+    :show-toggle-all="false"
+    :class="innerClass"
+    :optionValue="optionValue"
+    :optionLabel="optionLabel"
+    :maxSelectedLabels="3"
+  />
+  <!--
+    Note: It is required to set the id of this div to the FormKit node Id to allow the checkCustomInputs methods
+    in the validationUtils.ts file to scroll to this component when an error is detected. This is because the FormKit
+    List type does not create a wrapper component on its own.
+  -->
+  <div :id="formkitMultiSelectFormElementBindData?.node?.props?.id || undefined">
+    <FormKit
```

Comment:
Why do we even need this Formkit here if we don't use it to submit data?

Reply:
Yes you are right. I'll remove this input and add it if/when needed, no need for it to be confusing

Reply:
👍